package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "hubot-routines",
+  "version": "0.0.2",
+  "description": "functions for hubot-bot",
```

Comment:
Library containing the functions which could be useful in Hubot scripts development